src/lib/y2network/interface_config_builders/ctc.rb

```diff
@@ -25,10 +25,24 @@ module InterfaceConfigBuilders
     class Ctc < InterfaceConfigBuilder
       extend Forwardable
 
+      Yast.import "NetworkConfig"
+
       def initialize(config: nil)
         super(type: InterfaceType::CTC, config: config)
       end
 
+      def save
```

Comment:
Could we extend the unit test covering this ?

Reply:
yep